test/service/test_manifest.py

```diff
@@ -440,14 +440,37 @@ def test_manifest_zarr(self):
                 self.assertEqual(200, response.status_code)
                 lines = response.content.decode().splitlines()
                 file_prefix = 'output="587d74b4-1075-4bbf-b96a-4d1ede0481b2/'
+                file_location_prefix = f'url="{config.service_endpoint()}/repository/files'
                 curl_files = []
+                curl_download_urls = []
+                curl_aux_download_urls = []
                 for line in lines:
                     if line.startswith(file_prefix):
                         self.assertTrue(line.endswith('"'))
                         file_name = line[len(file_prefix):-1]
                         curl_files.append(file_name)
+                    elif line.startswith(file_location_prefix):
+                        self.assertTrue(line.endswith('"'))
+                        download_url = line[len(file_location_prefix):-1]
+                        url = furl(download_url)
+                        try:
+                            url.args['drsPath']
+                        except KeyError:
+                            curl_download_urls.append(url.url)
```

Comment:
No need to introduce an asymmetry between the branches by only collecting this attribute.
```suggestion
                            curl_download_urls.append(url)
```